### DISCUSSION TITLE
Exploring potential for Maker to directly publish to Nostr, & have Mostro participate only after offer is taken

### DISCUSSION THREAD

**Discussion by nobu-maeda** [MostroP2P/mostro#55] [Ideas]
Wondering if its possible for Makers to directly publish offers to Nostr relays. And only have Mostro participate as a coordinator / escrow only after a Taker decides to take the offer. This can improve censorship resistance.

Moved from [Issue 49](https://github.com/MostroP2P/mostro/issues/49)

**Comment by grunch** [comment]
I don't see it technically, do you have something in mind, how would be the order flow of this use case? with details please

**Reply by nobu-maeda** [reply]
I am updating n3xB proposal with some of my latest understanding, along with what I envision Mostro can tweak to make it more generic and open for compatible implementations, or alternative trade mechanics (on-chain, or bonding, for example). Will take a bit tho. I'll post back.

**Comment by grunch** [comment]
closing this discussion due inativity, in case you have a technical proposal for this please re-open it
